### PR TITLE
chore: release test version action

### PR DESCRIPTION
**Pull request by Llandy3d**
## Description

Adds a github action to build a test version of k6 studio without having to jump through hoops.
The action will require the user to specify a `version` to be used for that specific test release.

## Checklist

- [ ] I have performed a self-review of my code.
- [ ] I have added tests for my changes.
- [ ] I have run linter locally (`npm run lint`) and all checks pass.
- [ ] I have run tests locally (`npm test`) and all tests pass.
- [ ] I have commented on my code, particularly in hard-to-understand areas.

### CODE REVIEW

File: .github/workflows/release-test-version.yml [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+        platform: [macos-latest, windows-latest, ubuntu-20.04]
+
+    runs-on: ${{ matrix.platform }}
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Set version for test release
+        with:
+          version: ${{ inputs.version }}
+        run: python update_version.py
```
Comment (going-confetti) [inline .github/workflows/release-test-version.yml:32]: Will it work if executed before `actions/setup-python@v4`?

File: .github/workflows/release-test-version.yml [hunk 2]
```diff
@@ -0,0 +1,132 @@
+name: 'Manual Release Test Version'
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: "Specify a version for the test release (ex. test-browser)"
+        required: true
+        type: string
+        default: "test-version"
```
Comment (going-confetti) [inline .github/workflows/release-test-version.yml:10]: IMO it's better to ask for a suffix that will then be appended to the current version. This way it will be clearer which version we're branching off of. Not a strong opinion though.

File: .github/workflows/release-test-version.yml [hunk 3]
```diff
@@ -0,0 +1,132 @@
+name: 'Manual Release Test Version'
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: "Specify a version for the test release (ex. test-browser)"
+        required: true
+        type: string
```
Comment (going-confetti) [inline .github/workflows/release-test-version.yml:9]: Do you know if the input string is trimmed by default? Just to make sure there won't be version ` `

File: .github/workflows/release-test-version.yml (hunk 2 above)
Comment (Llandy3d) [reply .github/workflows/release-test-version.yml:10]: We can probably debate it in the future although it doesn't feel mandatory to have the current version there in favor of a more expressive name 🤔

File: .github/workflows/release-test-version.yml (hunk 3 above)
Comment (Llandy3d) [reply .github/workflows/release-test-version.yml:9]: I don't think it's trimmed since the default value if not set like we do we would be `''`, feels like a small edge case not too important because the most common case of forgetting to set it would be that the default applies of `test-version`

File: .github/workflows/release-test-version.yml (hunk 2 above)
Comment (going-confetti) [reply .github/workflows/release-test-version.yml:10]: One doesn't negate the other: `1.1.0-test` is more expressive and has more info that `test-version`

Comment (Llandy3d) [reply .github/workflows/release-test-version.yml:10]: I was thinking specifically as browser as an example, it would remove the possibility of having a full custom name like it was used recently if we always append to the end 🤔 
But it could be a good default (ex. if the version field is not specified, we just append `test` to the version)

Comment (going-confetti) [reply .github/workflows/release-test-version.yml:10]: > it would remove the possibility of having a full custom name like it was used recently if we always append to the end

Do we need this?
In my opinion. having a version for the browser builds makes a lot of sense: the browser branch has been frequently synced with main, and having a version there makes it easier to distinguish different builds

**Review by going-confetti — APPROVED** [review]
